### PR TITLE
Pluralize your application title wherever the place in the app

### PR DESCRIPTION
**Pull request by tomas-stefano**
## Context

This PR adds the pluralization for "Your application" / "Your applications" in the app

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -65,6 +65,11 @@ en:
   realistic_job_previews:
     url_apply_again: https://teacherselect.qualtrics.com/jfe/form/SV_4VHMkFk6TeonVeS
   page_titles:
+    continuous_applications:
+      your_application:
+        zero: Your application
+        one: Your application
+        other: Your applications
```
Comment (avinhurry) [inline config/locales/en.yml:72]:
```suggestion
        many: Your applications
```

**Review by avinhurry — APPROVED** [review]
Looks fine to me, left a suggestion in line.

Do you think it's worth testing one application?

File: config/locales/en.yml (hunk 1 above)
Comment (tomas-stefano) [reply config/locales/en.yml:72]: Just to be documented many got discarded from Rails. Now we need to use other key. :)

**Comment by tomas-stefano** [comment]
> Do you think it's worth testing one application?
It is done on the spec already :)

**Review by JR-G — APPROVED** [review]

**Pull request merged**